<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UTP Student Manager - Ventana Administrador</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Img/icono.jpg</normaloff>Img/icono.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="subtitulo_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>503</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
background: none;
color: #E0E0E0;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Accede a información detallada sobre cada alumno, incluyendo calificaciones, asistencia y matrícula&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="banner">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1001</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Img/imgPrincipal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logoUTP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Proyecto/Gestion_Alumnos/UI/img/icono.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo_2">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>483</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
background: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Los administradores pueden gestionar pagos, inscripciones y realizar un seguimiento de los estados financieros de los estudiantes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>493</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
background: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Genera reportes completos sobre el rendimiento académico, pagos, y más, para una gestión eficiente y organizada.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titulo1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>443</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
background: none</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Gestiona el progreso académico de tus estudiantes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titulo2">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>439</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
background: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Facilita la gestión de pagos y matrículas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titulo3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>438</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
background: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Accede a informes y reportes personalizados&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="icon1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>393</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Img/iconoPrincipal1_fondoBlanco.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="icon2">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>394</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Img/iconoPrincipal2_fondoBlanco.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="icon3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Img/iconoPrincipal3_fondoBlanco.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>363</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 12, 47);
border-radius: 30px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>363</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 30px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>363</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 12, 47);
border-radius: 30px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>subtitulo_1</zorder>
   <zorder>banner</zorder>
   <zorder>logoUTP</zorder>
   <zorder>subtitulo_2</zorder>
   <zorder>subtitulo_3</zorder>
   <zorder>titulo1</zorder>
   <zorder>titulo2</zorder>
   <zorder>titulo3</zorder>
   <zorder>icon1</zorder>
   <zorder>icon2</zorder>
   <zorder>icon3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEstudiantes">
    <property name="title">
     <string>Estudiantes</string>
    </property>
    <addaction name="menu_agregarEstudiantes"/>
    <addaction name="menu_MatricularEstudiante"/>
    <addaction name="menu_GestionarPagos"/>
   </widget>
   <widget class="QMenu" name="menuGesti_n_Acad_mica">
    <property name="title">
     <string>Gestión Académica</string>
    </property>
    <addaction name="menu_RegistrarCalificaciones"/>
    <addaction name="menu_ListaDocentes"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="menu_GenerarReporteAlumno"/>
    <addaction name="menu_GenerarReporteCalificaciones"/>
    <addaction name="menu_GenerarReportePagos"/>
   </widget>
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>Configuración</string>
    </property>
    <addaction name="menu_AdministradorUsuarios"/>
    <addaction name="menu_CerrarSesion"/>
   </widget>
   <addaction name="menuEstudiantes"/>
   <addaction name="menuGesti_n_Acad_mica"/>
   <addaction name="menuReportes"/>
   <addaction name="menuConfiguraci_n"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_VerPerfil">
   <property name="icon">
    <iconset>
     <normaloff>Img/perfil.png</normaloff>Img/perfil.png</iconset>
   </property>
   <property name="text">
    <string>Ver perfil</string>
   </property>
  </action>
  <action name="menu_AdministradorUsuarios">
   <property name="icon">
    <iconset>
     <normaloff>Img/usuarios.png</normaloff>Img/usuarios.png</iconset>
   </property>
   <property name="text">
    <string>Administrador usuarios</string>
   </property>
  </action>
  <action name="menu_GenerarReporteAlumno">
   <property name="icon">
    <iconset>
     <normaloff>Img/reporteAlumno.png</normaloff>Img/reporteAlumno.png</iconset>
   </property>
   <property name="text">
    <string>Generar Reporte de Alumno</string>
   </property>
  </action>
  <action name="menu_GenerarReporteCalificaciones">
   <property name="icon">
    <iconset>
     <normaloff>Img/reporteCalificaciones.png</normaloff>Img/reporteCalificaciones.png</iconset>
   </property>
   <property name="text">
    <string>Generar Reporte de Calificaciones</string>
   </property>
  </action>
  <action name="menu_GenerarReportePagos">
   <property name="icon">
    <iconset>
     <normaloff>Img/reportePagos.png</normaloff>Img/reportePagos.png</iconset>
   </property>
   <property name="text">
    <string>Generar Reporte de Pagos</string>
   </property>
  </action>
  <action name="menu_RegistrarCalificaciones">
   <property name="icon">
    <iconset>
     <normaloff>Img/boleta-de-calificaciones.png</normaloff>Img/boleta-de-calificaciones.png</iconset>
   </property>
   <property name="text">
    <string>Registrar Calificaciones</string>
   </property>
  </action>
  <action name="menu_ListaDocentes">
   <property name="icon">
    <iconset>
     <normaloff>Img/profesor2.png</normaloff>Img/profesor2.png</iconset>
   </property>
   <property name="text">
    <string>Ver Lista de Docentes</string>
   </property>
  </action>
  <action name="menu_agregarEstudiantes">
   <property name="icon">
    <iconset>
     <normaloff>Img/listaAlumnos.png</normaloff>Img/listaAlumnos.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Estudiante</string>
   </property>
  </action>
  <action name="menu_GestionarPagos">
   <property name="icon">
    <iconset>
     <normaloff>Img/pagos.png</normaloff>Img/pagos.png</iconset>
   </property>
   <property name="text">
    <string>Gestionar Pagos</string>
   </property>
  </action>
  <action name="menu_CerrarSesion">
   <property name="icon">
    <iconset>
     <normaloff>Img/cerrarSesion.png</normaloff>Img/cerrarSesion.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
  </action>
  <action name="menu_MatricularEstudiante">
   <property name="icon">
    <iconset>
     <normaloff>Img/agregarAlumno.png</normaloff>Img/agregarAlumno.png</iconset>
   </property>
   <property name="text">
    <string>Matricular Estudiante</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
